# House Keeping Gene

 [Skip to Main content](https://www.sciencedirect.com/topics/medicine-and-dentistry/housekeeping-gene#main_content)[![Elsevier logo](https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/24/images/elsevier-non-solus-new-grey.svg)](https://www.sciencedirect.com/)

* [Journals & Books](https://www.sciencedirect.com/browse/journals-and-books)
* * 
[Register](https://www.sciencedirect.com/user/register?targetURL=%2Ftopics%2Fmedicine-and-dentistry%2Fhousekeeping-gene)[Sign in](https://www.sciencedirect.com/user/login?targetURL=%2Ftopics%2Fmedicine-and-dentistry%2Fhousekeeping-gene&from=globalheader)

## Housekeeping Gene

Housekeeping genes are examples of regions in a genome that tend to be highly conserved and evolve slower than other genes such as the tissue-specific genes \(Zhang and Li, 2004\), mainly due to their roles in the maintenance of basic cellular functions and are essential for the existence of a cell.

From: [Encyclopedia of Bioinformatics and Computational Biology, 2019](https://www.sciencedirect.com/science/article/pii/B978012809633820126X)

### Related terms:

* [Enzyme](https://www.sciencedirect.com/topics/medicine-and-dentistry/enzyme)
* [Protein](https://www.sciencedirect.com/topics/medicine-and-dentistry/protein)
* [Multilocus Sequence Typing](https://www.sciencedirect.com/topics/medicine-and-dentistry/multilocus-sequence-typing)
* [Gene Expression](https://www.sciencedirect.com/topics/medicine-and-dentistry/gene-expression)
* [DNA](https://www.sciencedirect.com/topics/medicine-and-dentistry/dna)
* [RNA](https://www.sciencedirect.com/topics/medicine-and-dentistry/rna)
* [Polymerase Chain Reaction](https://www.sciencedirect.com/topics/medicine-and-dentistry/polymerase-chain-reaction)
* [Messenger RNA](https://www.sciencedirect.com/topics/medicine-and-dentistry/messenger-rna)

[View all Topics](https://www.sciencedirect.com/topics/index)Download as PDFSet alertAbout this page

### [Bacterial Pathogens](https://www.sciencedirect.com/science/article/pii/B9780120884834500081) <a id="tp-snippet-chp-title-B9780120884834500081"></a>

PAUL KEIM, in [Microbial Forensics](https://www.sciencedirect.com/book/9780120884834/microbial-forensics), 2005

#### MULTILOCUS SEQUENCE TYPING \(MLST\)

Housekeeping genes are essential, are found by definition in all bacteria, and evolve more rapidly than [16S rRNA](https://www.sciencedirect.com/topics/medicine-and-dentistry/rna-16s) genes. They are thus more discriminating regions. While there are hundreds or even a thousand of these genes available in any genome, researchers tend to pick a small number \(less than 10\) that are well distributed around the genome. Housekeeping gene regions of value can be studied across the target organism and its close relatives by designing “universal” PCR primers to relatively constant [nucleotide sequences](https://www.sciencedirect.com/topics/medicine-and-dentistry/nucleotide-sequence) to amplify more variable intervening regions. Some pathogens have been extensively studied in this fashion, but many others have not. Hence, forensic investigators will need to generate these data, and the value, or lack of value, in this MLST approach will remain unknown until the studies are completed. However, if an extensive MLST database exists for the pathogen under forensic investigation, this can be a very powerful means to identify the pathogen and subtype strains within the pathogen.[View chapter](https://www.sciencedirect.com/science/article/pii/B9780120884834500081)[Purchase book](https://www.elsevier.com/books/T/A/9780120884834)

### [New Approaches to Prokaryotic Systematics](https://www.sciencedirect.com/science/article/pii/S0580951714000178) <a id="tp-snippet-chp-title-S0580951714000178"></a>

Xiaoying Rong, Ying Huang, in [Methods in Microbiology](https://www.sciencedirect.com/bookseries/methods-in-microbiology), 2014

#### 2.2 Selection of gene loci

The [housekeeping genes](https://www.sciencedirect.com/topics/immunology-and-microbiology/housekeeping-gene) selected for MLSA analyses should be single copies, orthologous and ubiquitous among all of the sampled strains. They should also be highly conserved, have no [linkage disequilibrium](https://www.sciencedirect.com/topics/immunology-and-microbiology/gene-linkage-disequilibrium) on the chromosome but should contain sufficient varying [nucleotide](https://www.sciencedirect.com/topics/medicine-and-dentistry/nucleotide) positions to accurately establish relationships between closely related strains. To strike a balance among acceptable identification power, time and cost for strain typing, about five to seven housekeeping genes are commonly used. The five housekeeping genes that we have used in our [_Streptomyces_](https://www.sciencedirect.com/topics/immunology-and-microbiology/streptomyces) MLSA scheme are _atpD_ \(ATP synthase F1, _β_-subunit\), _gyrB_ \(DNA gyrase, B subunit\), _recA_ \(recombinase A\), _rpoB_ \(RNA polymerase, _β_-subunit\) and _trpB_ \(tryptophan synthase, _β_-subunit\); the physical distance between any two of these genes is greater than 30 kb in a single _Streptomyces_ genome, thereby avoiding hitchhiking effects \(Guo, Zheng, Rong, & Huang, 2008; Rong & Huang, 2012\). However, it is not uncommon to use up to 10 housekeeping genes, as exemplified in the case of the genus [_Nocardia_](https://www.sciencedirect.com/topics/immunology-and-microbiology/nocardia) where 14 protein-coding genes were examined \(Tamura et al., 2012\). Thus, both the number and type of housekeeping genes interrogated by MLSA may differ from genus to genus.[View chapter](https://www.sciencedirect.com/science/article/pii/S0580951714000178)[Purchase book](https://www.elsevier.com/books/T/A/9780128001769)

### [Functional Glycomics](https://www.sciencedirect.com/science/article/pii/S0076687910790042) <a id="tp-snippet-chp-title-S0076687910790042"></a>

Alison V. Nairn, ... Kelley W. Moremen, in [Methods in Enzymology](https://www.sciencedirect.com/bookseries/methods-in-enzymology), 2010

#### 2.7 Normalization gene selection

Several [housekeeping genes](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/housekeeping-gene) were analyzed to determine which gene had the most stable expression over the range of samples for a given study. [qRT-PCR](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/quantitative-reverse-transcription-polymerase-chain-reaction) reactions with cDNA templates from stem cell samples were assayed using several housekeeping genes to determine the variability of expression across all cell types. The gene with the lowest variation across all tissues was selected as the normalization gene for all samples. Alternatively, several software programs \(i.e., GeNorm and Normfinder; Andersen _et al._, 2004; Vandesompele _et al._, 2002\) are available for selection of normalization gene\(s\).[View chapter](https://www.sciencedirect.com/science/article/pii/S0076687910790042)[Purchase book](https://www.elsevier.com/books/T/A/9780123809971)

### [History, Science and Methods](https://www.sciencedirect.com/science/article/pii/B9780123786128001189) <a id="tp-snippet-chp-title-B9780123786128001189"></a>

T. Ramamurthy, G.B. Nair, in [Encyclopedia of Food Safety](https://www.sciencedirect.com/referencework/9780123786135/encyclopedia-of-food-safety), 2014

#### Variable Number of Tandem Repeat \(VNTR\) Loci

In housekeeping genes, regions known as VNTR are classified on the basis of their repeat units. These repeat loci are highly conserved and hence the discrimination power is more compared to that of [MLST](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/multilocus-sequence-typing). VNTR-based fingerprinting discriminated clinical O3:K6 strains from different outbreaks and sporadic cases in Tokyo during 1996–2003. [Multiple-locus VNTR analysis](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/multiple-loci-vntr-analysis) \(MLVA\) displayed between 2 and 15 alleles at each of 8 loci and shown to have high resolution and reproducibility for typing of _V. parahaemolyticus_. VNTR assay was applied to clinical and environmental _V. parahaemolyticus_ strains from Chile with 24 potential loci on both chromosomes. In this analysis, 59 MLVA groups were identified, providing differentiation among very closely related strains.[View chapter](https://www.sciencedirect.com/science/article/pii/B9780123786128001189)[Purchase book](https://www.elsevier.com/books/T/A/9780123786135)

### [The Formation of DNA Methylation Patterns and the Silencing of Genes](https://www.sciencedirect.com/science/article/pii/S0079660308602822) <a id="tp-snippet-chp-title-S0079660308602822"></a>

Jean-Pierre Jost, Alain Bruhat, in [Progress in Nucleic Acid Research and Molecular Biology](https://www.sciencedirect.com/bookseries/progress-in-nucleic-acid-research-and-molecular-biology), 1997

#### 1 CIS-ACTING REGULATORY ELEMENTS OF METHYLATION

Most [housekeeping genes](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/housekeeping-gene) have [CpG islands](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cpg-island) that are not normally methylated. For example, in the mouse [adenine](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/adenine) phosphoribosyl [transferase](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/transferase) \(_aprt_\) gene the CpG islands are totally free of [methylation](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/methylation), whereas the [flanking regions](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/dna-flanking-region) have methylated CpGs \(_94_\). In a transgenic mouse assay, deletion [mutagenesis](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/mutagenesis) of the Spl sites flanking the CpG islands of the _aprt_ gene initiated a _de novo_ methylation of the CpG islands. These experiments demonstrated that the peripherally located Spl sites were necessary to keep the _aprt_ CpG islands methylation free \(_94, 95_\). Though certain [cis-acting elements](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cis-acting-element) can effectively prevent the methylation of DNA, other sequences can enhance [DNA methylation](https://www.sciencedirect.com/topics/immunology-and-microbiology/dna-methylation). For example, Mummanemi _et al._ \(_96_\) showed conclusively that a cis-acting element located 1.3 kb upstream of the mouse _aprt_ gene acted as a signal for the methylation and epigenetic inactivation of the gene. During [X chromosome](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/x-chromosome) inactivation, specific sequences in the X inactivation center \(XIC\) may also play a fundamental role for the spreading of methylation from the XIC in both directions \(_97_\). In the above cases it is clear that cis control elements can prevent or give the signal for _de novo_ methylation. It is, however, not certain by which mechanism they function. Two possibilities, which are not mutually exclusive, should be considered: the structure of the DNA could either enhance or prevent the action of DNA [methyltransferase](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/methyltransferase), or there could be an indirect effect mediated by some specific RNA and proteins.[View chapter](https://www.sciencedirect.com/science/article/pii/S0079660308602822)[Purchase book](https://www.elsevier.com/books/T/A/9780125400572)

### [History, Science and Methods](https://www.sciencedirect.com/science/article/pii/B9780123786128001177) <a id="tp-snippet-chp-title-B9780123786128001177"></a>

T. Ramamurthy, G.B. Nair, in [Encyclopedia of Food Safety](https://www.sciencedirect.com/referencework/9780123786135/encyclopedia-of-food-safety), 2014

#### Variable Number of Tandem Repeat \(VNTR\) Loci

In many housekeeping genes, the DNA regions known as VNTR are cataloged on the basis of their repeat unit. VNTR is otherwise called [simple sequence repeats](https://www.sciencedirect.com/topics/medicine-and-dentistry/simple-sequence-repeat) . This [repetitive DNA](https://www.sciencedirect.com/topics/medicine-and-dentistry/repetitive-dna) contains monomeric sequences and arranged in a head-to-tail configuration. These repeat loci are highly conserved and hence the discrimination power is more compared with that of [MLST](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/multilocus-sequence-typing). Five VNTR loci of _V._ [_cholerae_](https://www.sciencedirect.com/topics/nursing-and-health-professions/cholera) strains collected between 1992 and 2007 from different areas in India showed that each VNTR locus was highly variable, with 5–19 alleles. The eBURST \(based upon related sequence types\) analysis of sequence types revealed four large groups of genetically related isolates with two groups containing O139 [serogroup](https://www.sciencedirect.com/topics/medicine-and-dentistry/serotype) and the other two groups including O1strains. VNTR also helped to track the spread of specific genotypes across time and space. Genetic relatedness of _V. cholerae_ collected from 2004–05 from Bangladesh showed minimal overlap in VNTR patterns between the two communities that was consistent with sequential, small outbreaks from local sources. VNTR-analysis of nontoxigenic _V. cholerae_ that caused outbreak in Rostov region in 2005 showed that they differed from previous outbreaks and formed separate group with strains isolated from patients, carriers, and environment. [Phylogenetic analysis](https://www.sciencedirect.com/topics/medicine-and-dentistry/phylogeny) of the combined VNTR data also showed a clear discrimination between the clinical O1 and O139 strains and the environmental isolates.[View chapter](https://www.sciencedirect.com/science/article/pii/B9780123786128001177)[Purchase book](https://www.elsevier.com/books/T/A/9780123786135)

### [Shigella and Shigellosis](https://www.sciencedirect.com/science/article/pii/B9780123971692000652) <a id="tp-snippet-chp-title-B9780123971692000652"></a>

Sophie Octavia, Ruiting Lan, in [Molecular Medical Microbiology \(Second Edition\)](https://www.sciencedirect.com/book/9780123971692/molecular-medical-microbiology), 2015

#### Phylogenetic Classification

Sequencing of eight housekeeping genes from representative strains from 46 [serotypes](https://www.sciencedirect.com/topics/medicine-and-dentistry/serotype) of [_Shigella_](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/shigella) showed that they were grouped into three major clusters \(Fig. 65.1\) \[6\]. Cluster 1 contained the majority of _S. boydii_ and _S._ [_dysenteriae_](https://www.sciencedirect.com/topics/medicine-and-dentistry/dysentery) strains \(_S. boydii_ B1–4, B6, B8, B10, B14, B18, and _S. dysenteriae_ D3–7, D9, D11–D13\) as well as _S. flexneri_ 6 and 6A. Cluster 2 consisted of seven _S. boydii_ strains \(B5, B7, B9, B11, B15, B16 and B17\) and one _S. dysenteriae_ strain \(D2\). Cluster 3 contained one _S. boydii strain_ \(B12\) and _S. flexneri_ serotypes 1–5 as well as their subtypes. _S. sonnei_ and three _S. dysenteriae_ strains \(D1, D8 and D10\) did not fall into any of the three major _Shigella_ clusters but were clearly within _E. coli_. One _S. boydii_ strain \(B13\) also formed an outlier and was distantly related to _E. coli_. _S. boydii_ B13 appears to have evolved from an _E. coli_-like ancestor approximately 28 million years ago and forms a lineage with [_Escherichia_](https://www.sciencedirect.com/topics/medicine-and-dentistry/escherichia) _albertii_ \[7\]. However, Hyma et al. \[7\] also showed that one group of B13 is located within _E. coli_ suggesting two distinct groups of B13. The sequence data also suggest that the independent _Shigella_ lineages originated from _E. coli_ 35 000–270 000 years ago through independent [convergent evolution](https://www.sciencedirect.com/topics/medicine-and-dentistry/convergent-evolution) \[6\]. The _Shigella_ and _E. coli_ relationship was further confirmed by other studies with sequencing of more housekeeping genes \[121,122\] and by genome sequences \[123,124\].![](https://ars.els-cdn.com/content/image/3-s2.0-B9780123971692000652-f65-01-9780123971692.jpg)[Sign in to download full-size image](https://www.sciencedirect.com/user/login?returnURL=https%3A%2F%2Fwww.sciencedirect.com%2Ftopics%2Fmedicine-and-dentistry%2Fhousekeeping-gene)

Figure 65.1. Phylogenetic relationships of _Shigella_ and _E. coli_ based on housekeeping genes \[6\]. _Shigella_ strains were differentiated into three clusters, each of which includes strains from more than one _Shigella_ species and outliers \(_S. dysenteriae_ 1, 8, 10 and _S. boydii_ 13 and _S. sonnei_\). _Salmonella typhimurium_ LT2 was used as an outgroup. All other strains are _E. coli_ strains.Modified from \[9\].[View chapter](https://www.sciencedirect.com/science/article/pii/B9780123971692000652)[Purchase book](https://www.elsevier.com/books/T/A/9780123971692)

### [The Role of Moonlighting GAPDH in Cell Proliferation](https://www.sciencedirect.com/science/article/pii/B9780128098523000017) <a id="tp-snippet-chp-title-B9780128098523000017"></a>

Michael A. Sirover, in [Glyceraldehyde-3-phosphate Dehydrogenase \(GAPDH\)](https://www.sciencedirect.com/book/9780128098523/glyceraldehyde-3-phosphate-dehydrogenase-gapdh), 2017

#### Abstract

The distinction between [housekeeping genes](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/housekeeping-gene) and those genes that are actively regulated may be defined by several criteria. Among these may be the dependence of the latter on the proliferative state of the cell, especially with respect to their [subcellular localization](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/subcellular-localization), transcription, [biosynthesis](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/anabolism), and interactions with other cellular constituents. Recent studies suggest that moonlighting GAPDH exhibits defined changes in its expression as a function of [cell proliferation](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cell-proliferation). These include its cytosolic localization in noncycling cells as contrasted with its perinuclear or nuclear localization in cycling cells; its cytoplasmic relocalization as cell proliferation diminishes; enhancement of both GAPDH mRNA and protein levels in cycling cells; the physical association of GAPDH with replicating DNA; and its dissociation from the latter as cell growth diminishes. Further analysis suggests not only that GAPDH may regulate cell cycle transition but also the initiation of cell senescence. In toto, these findings suggest a complex role for moonlighting GAPDH during cell proliferation.[View chapter](https://www.sciencedirect.com/science/article/pii/B9780128098523000017)[Purchase book](https://www.elsevier.com/books/T/A/9780128098523)

### [Zebrafish offer aquaculture research their services](https://www.sciencedirect.com/science/article/pii/B978012801418900007X) <a id="tp-snippet-chp-title-B978012801418900007X"></a>

Peter Aleström, Hanne C. Winther-Larsen, in [Genomics in Aquaculture](https://www.sciencedirect.com/book/9780128014189/genomics-in-aquaculture), 2016

#### Expressing the genome—the transcriptome defines all transcripts produced

Although many “house-keeping” genes and functions are common between fish, there is one distinct [transcriptome](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/transcriptome) for each cell type and developmental stage with a specific spatial and temporal expression pattern in all of them. Also, environmental factors like drugs, pollutants, pathogens etc. can induce changes in a transcriptome and these changes can in principle be mapped to deciphering of modes of action for each effector. In general, the total transcriptome is composed of a vast number of different types of RNA molecules, which can be subdivided into protein coding \(mRNAs\) and noncoding RNAs \(ncRNAs\). With 25,642 protein coding genes and 57,369 annotated gene transcripts \(mRNAs\) in GRCz10, there is an average of 2.2 isoforms transcribed from each gene, with the isoform diversity arising from the use of alternative start, stop, and or splicing choices. Different isoforms from one gene may express different proteins, or proteins with altered composition of functional domains, for example, generated from exon skipping or [intron retention](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/intron-retention) events \(Aanes et al., 2013; Aanes et al., 2011\). In an attempt to increase the annotation quality of zebrafish genomics datasets, five tissues and seven developmental stages were selected for [RNA sequencing](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/rna-sequence) of the protein coding transcriptome \(Collins et al., 2012\). The maternal transcriptome stored in the female [germline](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/germ-cells) controls the early embryo onset of the zygotic genome \(maternal-zygotic transition; MZT\) and the concomitant initiation of differentiation into the three [germ layers](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/germ-layer) endo-, ecto-, and [mesoderm](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/mesoderm) in the gastrulating embryo. In zebrafish there are &gt;10,000 maternal mRNAs of which 70% are dormant in the egg and becomes translationally activated by cytoplasmic [polyadenylation](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/polyadenylation) between the 1-cell and 16-cell stage \(Aanes et al., 2011\). The old paradigm for MZT occurring during the [midblastula](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/midblastula) transition at the 10th symmetric cell division around 3hpf is now being modified with more sensitive methods used and reveals that the activation of the zygotic genome rather takes place more gradually from around 2hpf and the 64- to 128-cell state \(Heyn et al., 2014\). In the maternal transcriptome, many genes are expressed as multiple isoforms which further add to its complexity \(Aanes et al., 2013\).

To serve as a model for other important but less well-characterized [aquaculture](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/aquaculture) relevant species, such as [turbot](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/turbot), catfish, and [salmonids](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/salmonid), a good annotation and clear understanding of the genetic and epigenetic control of the biology is essential. Experimental [functional genomics](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/functional-genomics) is needed to link genotypes and phenotypes. The zebrafish mutation project at the Welcome Trust Sanger Institute aims at establishing a resource consisting of a complete library of zebrafish mutants in which each cloned line of fish has one [gene knockout](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/gene-knockout) mutation, together making up the zebrafish [_phenome_](https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/phenome) \(Kettleborough et al., 2013\) and see http://www.sanger.ac.uk/sanger/Zebrafish\_Zmpbrowse\). In conclusion, extensive annotation of a genome is needed in order to fully make use of its encoded information for depicting its translation into functional terms. Hence deep sequencing of model organisms plays an important role to bridge the resolution of genomes to applications via comparative genomics.[View chapter](https://www.sciencedirect.com/science/article/pii/B978012801418900007X)[Purchase book](https://www.elsevier.com/books/T/A/9780128014189)

### [Cytogenetic and Molecular Pathology of Pediatric Cancer](https://www.sciencedirect.com/science/article/pii/B9781416034315000054) <a id="tp-snippet-chp-title-B9781416034315000054"></a>

Maureen J. O'Sullivan, ... Carolyn A. Felix, in [Oncology of Infancy and Childhood](https://www.sciencedirect.com/book/9781416034315/oncology-of-infancy-and-childhood), 2009

#### Quantitative Polymerase Chain Reaction

Simultaneous PCR of a constitutively expressed housekeeping gene provides an internal control. A standard positive control may be used as a comparator across a series of cases to rank these in a semiquantitative approach.53 Use of quantitative PCR methods can provide precise information about the actual copy number of oncogene transcripts \(gene expression level\) by qRT-PCR or gene copy number when genomic DNA is used as template material. Quantitative PCR has advantages of increased reliability and reproducibility over conventional PCR. Because it provides data accrued during the entire cycle, rather than just end point analysis, as in standard PCR, it will discriminate between different assays quantitatively rather than based on product size differences. Additionally, the graphic readout of accumulating fluorescence generated within each tube or well during the reaction is instantaneous \(real time\) and therefore no delaying additional step for product analysis is required. In conventional PCR, the amplification products must typically be evaluated by gel electrophoresis. Therefore, the assay turnaround time is minimized in qPCR, so that a complete analysis can be performed in only a few hours. qPCR is more sensitive, with an ability to detect twofold changes in expression level as opposed to a 10-fold cutoff for standard PCR assays. If desired, the products can still be subjected to gel electrophoresis—for example, to evaluate product size. Various forms of qPCR are available, including: \(1\) an SYBR green-based method, detecting double-stranded DNA product accumulation by binding to the minor groove of double-stranded DNA nonspecifically and thus being more readily subject to false positivity because of contamination or detection of primer-dimer; and \(2\) TaqMan methodology, based on highly specific primer-probe combinations. In TaqMan qPCR, a probe labeled with a fluorescent dye at one end \(5′\) and a quencher at the other \(3′\) is designed to bind within the sequence amplified by qPCR. In the unbound state, the proximity of quencher to the fluorescent dye inhibits detection of the fluorescence through Foerster resonance energy transfer \(FRET\). Once the probe has bound within the amplified sequence, 5′ exonuclease activity of the polymerase enzyme leads to probe cleavage and dissociation of fluorescent dye and quencher, with ensuing accumulation of fluorescence proportional to product amplification \(strictly probe cleavage\). Multiplexing of reactions is possible with TaqMan but not SYBR green technology.[View chapter](https://www.sciencedirect.com/science/article/pii/B9781416034315000054)[Purchase book](https://www.elsevier.com/books/T/A/9781416034315)

### Recommended publications

* [![Publication cover](https://ars.els-cdn.com/content/image/1-s2.0-S0002944020X00071-cov150h.gif)](https://www.sciencedirect.com/journal/the-american-journal-of-pathology)[**The American Journal of Pathology**](https://www.sciencedirect.com/journal/the-american-journal-of-pathology)Journal
* [![Publication cover](https://ars.els-cdn.com/content/image/1-s2.0-S0091674920X00078-cov150h.gif)](https://www.sciencedirect.com/journal/journal-of-allergy-and-clinical-immunology)[**Journal of Allergy and Clinical Immunology**](https://www.sciencedirect.com/journal/journal-of-allergy-and-clinical-immunology)Journal
* [![Publication cover](https://ars.els-cdn.com/content/image/1-s2.0-S1567576920X00075-cov150h.gif)](https://www.sciencedirect.com/journal/international-immunopharmacology)[**International Immunopharmacology**](https://www.sciencedirect.com/journal/international-immunopharmacology)Journal
* [![Publication cover](https://ars.els-cdn.com/content/image/1-s2.0-S0022202X20X0008X-cov150h.gif)](https://www.sciencedirect.com/journal/journal-of-investigative-dermatology)[**Journal of Investigative Dermatology**](https://www.sciencedirect.com/journal/journal-of-investigative-dermatology)Journal

[Browse books and journals](https://www.sciencedirect.com/browse/journals-and-books)[![Elsevier logo](https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/47/images/elsevier-non-solus-new-with-wordmark.svg)](https://www.elsevier.com/)

* [About ScienceDirect](https://www.elsevier.com/solutions/sciencedirect)
* [Remote access](https://www.sciencedirect.com/customer/authenticate/manra)
* [Shopping cart](https://sd-cart.elsevier.com/?)
* [Advertise](http://elsmediakits.com/)
* [Contact and support](https://service.elsevier.com/app/contact/supporthub/sciencedirect/)
* [Terms and conditions](https://www.elsevier.com/legal/elsevier-website-terms-and-conditions)
* [Privacy policy](https://www.elsevier.com/legal/privacy-policy)

We use cookies to help provide and enhance our service and tailor content and ads. By continuing you agree to the [**use of cookies**](https://www.sciencedirect.com/legal/use-of-cookies).

Copyright © 2021 Elsevier B.V. or its licensors or contributors. ScienceDirect ® is a registered trademark of Elsevier B.V.[![RELX group home page](https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/60/images/logo-relx-tm.svg)](https://www.relx.com/)

